iot/http_example/cloudiot_http_example.js

```diff
@@ -16,7 +16,7 @@
 // [START iot_http_includes]
 const fs = require('fs');
 const jwt = require('jsonwebtoken');
-const request = require('retry-request', {request: require('request')});
```

Comment:
I am pretty sure Gus moved these samples all over to here:
https://github.com/googleapis/nodejs-iot/blob/master/samples/http_example/cloudiot_http_example.js

Because I remember making a very similar PR 😆 Not blocking this PR, but it looks like we need to reconcile the samples left behind here with the ones in the client lib repo.

Reply:
ohhh, let me check the region tags and delete the duplicate samples 😃